WordPress/Classes/ViewRelated/Me/All Domains/Views/AllDomainsListViewController.swift

```diff
@@ -52,7 +54,13 @@ final class AllDomainsListViewController: UIViewController {
         self.viewModel.addDomainAction = { [weak self] in
             self?.navigateToAddDomain()
         }
-        self.title = Strings.title
+        if isNavigatingFromUIKit {
```

Comment:
The necessity for this is explained [here](https://stackoverflow.com/a/66224181).